.github/workflows/test-deploy.yml

```diff
@@ -0,0 +1,24 @@
+name: Test deployment
```

Comment:
This doesn't actually do any deploying to a preview site right? Maybe we should rename file/job schema to test build

Reply:
Correct! Good thought though, probably better with `Test Build` or something